model/user.js

```diff
@@ -1,15 +1,24 @@
 const mongoose = require("mongoose");
 const parameters = require("../utils/parameters.js");
 
+
+
 const UserSchema = mongoose.Schema({
     facebook: {
         id: String,
         token: String,
         email: String,
         name: String
-    }
+    },
     isAdmin : { type : Boolean , default : false } ,
-    accountBalance : Number ,
+    accountBalance : { type : Number , default : parameters.defaultbal},
+    stockShorted :[{
+        TotalPrice : Number,
+        TotalStock : Number
```

Comment:
what's the need for price? the price can be accessed from the company id of the company from which the transaction will be made i guess. lets just keep one key, `quantity : Number`

Reply:
price keeps changing so when a person covers the share we can access the price at which they were shorted and can calculate appropriately the profit of user